tools/masspix/.shed.yml

```diff
@@ -0,0 +1,11 @@
+name: massPix
+categories: 
+ - Metabolomics
+ - Lipidomics
+ - Mass Spectrometry Image Processing
+description: 'Processes high resolution mass spectrometry imaging data'
+homepage_url: https://github.com/hallz/massPix
+long_description: 'Processes high resolution mass spectrometry imaging data,  
+  performs multivariate statistics (PCA, clustering) and lipid identification.'
+owner: Wanchang Lin, Zoe Hall, Julian Griffin
```

Comment:
@lecorguille what is the global share TS owner that is used by this repo?

Reply:
So far, workflow4metabolomics
But it can be discused if it's a problem

Reply:
@wanchanglin can you update the owner in both shed.yml files to "workflow4metabolomics". This is the toolshed owner ...